tests/unit/fixtures/trendingFixtures.py

```diff
@@ -0,0 +1,47 @@
+import ROOT
+import pytest
+
+from overwatch.processing.trending.constants import EXTENSION, ENTRIES
+from overwatch.processing.trending.objects.mean import MeanTrending
+
+
+@pytest.fixture
+def trendingArgs():
+    parameters = {
+        'test': True, ENTRIES: 20, EXTENSION: 'png',
+    }
+    yield ["name", "desc", ["h1", "h2"], 'TST', parameters]
+
+
+@pytest.fixture
+def infoArgs():
+    yield ["name", "desc", ["hist1", "hist2"], MeanTrending]
+
+
+class Histogram:
```

Comment:
Please inherit from `object` for python 2 compatibility.
```suggestion
class Histogram(object):
```